scripts-server/restore.sh

```diff
@@ -4,6 +4,27 @@
 # Before executing make sure nobody is accessing the databases, i.e. stop all services except for the actual databases
 
 postgres_hosts=(postgres-core postgres-oms-statutory postgres-oms-events)
+#postgres_hosts=($2) #FIXME still to test
+
+#to get postgres version programmatically (wip): docker ps | grep postgres | awk '{print $2}'
+# NB weak because every ms may have a different version of postgres
```

Comment:
you are suggesting to also update all the postgres in the ms? I would not do that, it takes longer (more PRs). in that regard, I want to explore the monorepo way, because i also have the upcoming upgrade to traefik2 that would use a shortcut..!

Reply:
If you can't find it on Slack, this is an article I found that might be a nice way to convert to monorepo; https://x3ro.de/integrating-a-submodule-into-the-parent-repository/

Reply:
This isn't going to happen in this pr anyway, can you solve the conversation and approve?